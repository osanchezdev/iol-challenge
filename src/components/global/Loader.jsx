import React from 'react';
import {Spinner} from 'react-bootstrap';

import '../../styles/components/global/Loader.scss';

const Loader = () => {
  return (
    <div className="loader--wrapper">
      <Spinner animation="border" variant="light" />
    </div>
  );
};

export default Loader;
