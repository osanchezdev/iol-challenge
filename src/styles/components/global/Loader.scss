.loader--wrapper {
  min-height: 500px;
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
}
