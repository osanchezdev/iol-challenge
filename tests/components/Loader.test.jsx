import * as React from 'react';
import {shallow} from 'enzyme';
import Loader from '../../src/components/global/Loader';

describe('Tests on <Loader />', () => {
  test('v must be rendered correctly', () => {
    const wrapper = shallow(<Loader />);

    expect(wrapper).toMatchSnapshot();
  });
});
