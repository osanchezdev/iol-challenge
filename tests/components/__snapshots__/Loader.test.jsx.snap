// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <Loader /> <Loader /> must be rendered correctly 1`] = `
<div
  className="loader--wrapper"
>
  <Spinner
    animation="border"
    variant="light"
  />
</div>
`;
